docs/topics/js/js-project-setup.md

```diff
@@ -249,9 +251,16 @@ dependencies {
 </tab>
 </tabs>
 
-The plugin uses the [Yarn](https://yarnpkg.com/lang/en/) package manager to download and install npm dependencies.
-It works out of the box without additional configuration, but you can tune it to specific needs.
-Learn how to [configure Yarn in Kotlin Multiplatform Gradle plugin](#yarn).
+By default, the plugin uses the [Yarn](https://yarnpkg.com/lang/en/) package manager to download and install npm dependencies.
+It works out of the box without additional configuration, but you can [tune it to specific needs](#yarn).
+
+You can also use [npm](https://www.npmjs.com/) as your package manager instead. This makes the overall setup of your
+project easier, as you have one less tool to manage. To use npm as your package manager, in your `gradle.properties`
```

Comment:
From context, it's not evident to me why npm is easier and how there is one less tool to manage if we use one package manager instead of the other. Can we summarize the reasoning here?